orchestrators/FinanceOperations/tests/Unit/Rules/SubledgerClosedRuleTest.php

```
@@ -4,197 +4,98 @@
-            public function getPeriodId(): string {
-                return '2026-01';
-            }
-            public function getSubledgerType(): string {
-                return 'AR';
-            }
-        };
+        $rule = new SubledgerClosedRule($this->periodManager(isClosed: true));
 
-        $result = $rule->check($context);
+        $result = $rule->check(
+            RuleContext::forSubledgerClosure('', '2026-01', 'AR')
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\FinanceOperations\DTOs\RuleContext' in method 'testMissingTenantIdFailsValidation'. (undefined)

(StaticAccess)